### PR TITLE
Homework3: Aleksandr Brovko 

### PR DESCRIPTION
**Pull request by JanGonza**

### CODE REVIEW

File: src/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+""" FizzBuzz
+Напишите программу, которая печатает цифры от 1 до 100, но вместо чисел,
+кратных 3 пишет Fizz, вместо чисел кратный 5 пишет Buzz,
+а вместо чисел одновременно кратных и 3 и 5 - FizzBuzz
+"""
+
+for number in range(1, 101):
+    if number % 3 == 0 == number % 5:
```
Comment (belovmd) [inline src/FizzBuzz.py]: looks completely unclear

File: src/Сhocolate.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+if 1 <= k <= (n * m) - 2:
+    print("За некоторое количество разломов можно отломать {} кусков."
+          .format(k))
+else:
+    print("За некоторое количество разломов нельзя отломать {} кусков."
+          .format(k))
+
+"""Можно ли отломить от шоколадки ровно k долек с помощью t разломов
+
+Если k / 2 ** 7 <= 1 то можно получить k долек за t разломов.
```
Comment (belovmd) [inline src/Сhocolate.py]: why 7? missed solution description

**Review by belovmd — CHANGES_REQUESTED** [review]
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-python-spring/blob/master/homework_review/Homework3.md):
- 6
- 12
- 11
- 13

File: src/Сhocolate.py (hunk 2 above)
Comment (JanGonza) [reply src/Сhocolate.py]: Fixed
k / 2 ** t <= 1

File: src/FizzBuzz.py (hunk 1 above)
Comment (JanGonza) [reply src/FizzBuzz.py]: Fixed
if number % 15 == 0:

**Review by belovmd — COMMENTED** [review]
9,5

**Pull request closed without merging**